overlord/snapstate/autorefresh.go

```diff
@@ -406,3 +456,14 @@ func getTime(st *state.State, timeKey string) (time.Time, error) {
 	}
 	return t1, nil
 }
+
+func canRefreshOnMeteredConnection(st *state.State) (bool, error) {
```

Comment:
refreshMetered as a function name would be easier to read and understand. It might even return the actual value, and leave the logic for the main function.